### DISCUSSION TITLE
Priority inversion when loading url in global queue.

### DISCUSSION THREAD

**Discussion by wsdwsd08290829** [airbnb/lottie-ios#2463] [General]
There is report from system tools that using global queue could case priority inversion:  https://github.com/hansemannn/Onboard-Lottie/blob/master/Demo/Carthage/Checkouts/lottie-ios/lottie-ios/Classes/Private/LOTAnimationView.m#L68

Thus we propose to add api so users can provide a lower priority queue instead thanks.

**Comment by calda** [comment]
The code you linked looks like very only Objective-C code that no longer exists in recent versions of Lottie.
